Formula/pygobject3.rb

```diff
@@ -16,22 +15,32 @@ class Pygobject3 < Formula
   option "without-python", "Build without python3 support"
   option "with-python@2", "Build with python2 support"
 
-  depends_on "pkg-config" => :build
-  depends_on "libffi" => :optional
   depends_on "glib"
+  depends_on "gobject-introspection"
+  depends_on "libffi" => :optional
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
```

Comment:
Deps are structured by type & then alphabetically within those types, which I appreciate may be confusing. Essentially `:build` deps first, then mandatory dependencies, then recommended dependencies, then optional dependencies, then `if build.with?` dependencies.